gcp/README.md

```diff
@@ -673,3 +673,55 @@ resolved/update is done.
 If this downtime will be disruptive to your work ("We're giving a demo for the
 Prime Minister at that time!"), please let me know immediately.
 ```
+
+#### External backups
+
+##### Setup.
+
+Create a Terragrunt definition like `gcp/live/prd/k8s/kube-system/backup-exporter/terraform.tfvars` inside the exekube project. This file contains 3 variables:
```

Comment:
I don't understand why these instructions are necessary. Are there special steps here that we wouldn't follow for any module? If not, I would remove this section.

Reply:
I have removed them ( https://github.com/gpii-ops/gpii-infra/pull/432/commits/38542c9230965b1c36279bdf603dbb40192b1a3e ) and added them as a comment in the terraform.tfvars https://github.com/gpii-ops/gpii-infra/pull/431/commits/39041e8799492587fa40d22820f1b0cd5546c6ff